src/script.js

```diff
@@ -222,6 +222,24 @@ export function getLocale() : LocaleType {
         if (COUNTRY_LANGS.hasOwnProperty(country) && COUNTRY_LANGS[country].indexOf(lang) !== -1) {
             // $FlowFixMe
             return { country, lang };
+        } else {
+            // We will infer country from language if there is only one possible country match
+            const possibleCountries = Object.keys(COUNTRY_LANGS).filter((possibleCountry) => {
+                const languages = COUNTRY_LANGS[possibleCountry];
+
+                if (languages.length && languages.some(language => language === lang)) {
+                    return true;
+                }
+
+                return false;
+            });
+
+            const inferredCountry = COUNTRY[possibleCountries[0]];
```

Comment:
Can we do this lookup after checking `possibleCountries.length === 1`? Otherwise we're doing `COUNTRY[undefined]` which will technically work but feels error prone.